arm-operationalinsights/2015-03-20/swagger/OperationalInsights.json

```diff
@@ -0,0 +1,1110 @@
+        }
+      },
+      "description": "The saved search operation response."
+    },
+    "SearchError": {
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "The error type."
+        },
+        "message": {
+          "type": "string",
+          "description": "The error message."
+        }
+      },
+      "description": "Details for a search error."
+    },
+    "SearchResultsResponse": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The id of the search, which includes the full url."
+        },
+        "__metadata": {
+          "x-ms-client-name": "Metadata",
+          "$ref": "#/definitions/SearchMetadata",
+          "description": "The metadata from search results."
+        },
+        "value": {
+          "type": "array",
+          "items": {
+            "type": "object"
+          },
+          "description": "The array of result values."
+        },
+        "error": {
+          "$ref": "#/definitions/SearchError",
+          "description": "The error."
+        }
+      },
+      "description": "The get search result operation response."
+    },
+    "SearchSchemaValue": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name."
+        },
+        "displayName": {
+          "type": "string",
+          "description": "The display name."
+        },
+        "type": {
+          "type": "string",
+          "description": "The type."
+        },
+        "indexed": {
+          "type": "boolean",
+          "description": "The boolean that indicates the field is searchable as free text."
+        },
+        "stored": {
+          "type": "boolean",
+          "description": "The boolean that indicates whether or not the field is stored."
+        },
+        "facet": {
+          "type": "boolean",
+          "description": "The boolean that indicates whether or not the field is a facet."
+        },
+        "ownerType": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The array of workflows containing the field."
+        }
+      },
+      "required": [
+        "facet",
+        "stored",
+        "indexed"
+      ],
+      "description": "Value object for schema results."
+    },
+    "SearchGetSchemaResponse": {
+      "properties": {
+        "__metadata": {
+          "x-ms-client-name": "Metadata",
+          "$ref": "#/definitions/SearchMetadata",
+          "description": "The metadata from search results."
+        },
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SearchSchemaValue"
+          },
+          "description": "The array of result values."
+        }
+      },
+      "description": "The get schema operation response."
+    },
+    "SearchHighlight": {
+      "properties": {
```

Comment:
Just checked the implementation. For SearchHighlight, no properties are required. Empty object will just be ignored. Also, we don't define any patterns for the values of the properties here.